floris/simulation/turbine.py

```diff
@@ -215,14 +213,14 @@ def power(
         # type to the main thrust coefficient array
         p += power_interp[turb_type](rotor_effective_velocities) * (turbine_type_map == turb_type)
 
-    return p * ref_density_cp_ct
+    return p
```

Comment:
Can you explain this a bit more? Why does it work to remove this, and was it added somewhere else?
Also, it was not removed in the multidimensional turbine power function.

Reply:
This is the only line where the reference density is used in FLORIS (also in the multidimensional turbine power function). If it isn't needed here, then we can remove it entirely.

Reply:
Yeah, now that we use power rather than Cp, the air density is already built in to the power curve `power_interp`. I'm not exactly sure what the earlier reason was to leave density out of the `power_interp` when it was created from Cp values. However, the reference air density remains in FLORIS in various places, it's just been renamed to `ref_air_density`, so it won't show up in a search for `ref_density_cp_ct`. On the note of the multidimensional turbine power function, I'm removing that in #770 in favor of more streamlined `power()` (as well as `Ct()` and `axial_induction()`) that work for both "normal" and multidimensional turbines

Reply:
> However, the reference air density remains in FLORIS in various places, it's just been renamed to ref_air_density, so it won't show up in a search for ref_density_cp_ct.

I only see `ref_air_density` in places where the data is being constructed (primarily `Farm`) aside from places where the API hasn't been updated (calls to `turbine.power()` and `turbine.rotor_effective_velocity`).

Reply:
And thanks for clarifying why we can remove the air density here.

Reply:
Ah I see it's used in [turbine.rotor_effective_velocity](https://github.com/misi9170/floris/blob/v4-ms/power-replaces-cp/floris/simulation/turbine.py#L107). It's `ref_air_density` there rather than `ref_air_densities` so it didn't show up in a search. In any case, it's used in that function to allow for calculating power with a different air density than the one used to define the power curve.

Reply:
Good catch! We definitely will want to be able to simulate different air densities and have the power curve change automatically with it. I am guessing it's a scaling factor, so multiply power with `air_density / ref_air_density`, and then clip to the rated power to avoid producing more than rated?

Reply:
@Bartdoekemeijer similar to that. When the air density is different to the reference air density, we "slide" along the power curve in a similar way to how we compute power when there is a yaw misalignment---that is, for a given air density and rotor-averaged wind speed, we find an equivalent wind speed that pairs with the reference air density and sample the power curve at that wind speed instead.

Reply:
Makes a lot of sense, thanks for the clarification!